packages/test/mocha-test-setup/api-report/mocha-test-setup.api.md

```diff
@@ -4,8 +4,6 @@
 
 ```ts
 
-/// <reference types="mocha" />
```

Comment:
This seems like an unrelated change, and should probably be reverted (if possible).